src/main/java/mate/controller/driver/RegistrationDriverController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/add")
+public class RegistrationDriverController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate");
+    private static final String ADD_DRIVER_JSP = "/WEB-INF/views/driver/addDriver.jsp";
+    private final DriverService driverService =
+            (DriverService) INJECTOR.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher(ADD_DRIVER_JSP).forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String licenseNumber = req.getParameter("license_number");
+        Driver driver = new Driver();
+        driver.setName(name);
+        driver.setLicenseNumber(licenseNumber);
+        driverService.create(driver);
+    }
```

Comment:
After creating a new driver, you could redirect to the list of drivers or display a success message.